docs/setup-and-usage.md

```diff
@@ -109,7 +109,116 @@ cp configs/.env.example configs/.env # Edit the file after the copy.
 docker run -v $(pwd)/configs:/configs tellor/telliot:master mine
 ```
 
-## Run cli in mining mode with k8s
+## Run on Kubernetes with Helm Chart
```

Comment:
I was thinking to replace all k8s commands with the helm commands. 
Here I see that you added the helm commands, but haven't removed the kubectl commands.

We also need examples on how to update the secret and the config map.

Reply:
I don't think this has been addressed yet.

Reply:
Apologies, bit of commit confusion. Addressed.